### PR TITLE
Enable builds/ testing on ppc64le architecture

### PR DESCRIPTION
**Pull request by krishvoor**
This PR will allow to build container image
and allow e2e testing on ppc64le architecture

Signed-off-by: Krishna Harsha Voora <krishvoor@in.ibm.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,8 @@ COPY pkg/ pkg/
 COPY version/ version/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
```
Comment (eguzki) [inline Dockerfile:21]: Why not

```
RUN CGO_ENABLED=0 GOOS=linux GOARCH=`go env GOARCH` GO111MODULE=on go build -a -o manager main.go
```

File: Makefile [hunk 2]
```diff
@@ -66,7 +66,7 @@ TEST_E2E_PKGS = $(shell $(GO) list ./... | grep 'github.com/3scale/3scale-operat
 ENVTEST_ASSETS_DIR=$(PROJECT_PATH)/testbin
 test-e2e: generate fmt vet manifests
 	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
```
Comment (eguzki) [inline Makefile:69]: `controller-runtime v0.6.3` is being used when compiling the operator. 

Why should we change the version?

**Review by eguzki — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (krishvoor) [reply Dockerfile:21]: Makes sense, will update.

File: Makefile (hunk 2 above)
Comment (krishvoor) [reply Makefile:69]: `controller-runtime v0.6.3` does not contain ppc64le compatible `kubebuilder-tools`.
Bumping up the `controller-runtime v0.8.0` will allow `test-e2e` to be executed on ppc64le as well.

**Review by eguzki — APPROVED** [review]

**Pull request merged**